ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/DOWNLINK_IPV4_POOL_TESTS.yml

```diff
@@ -0,0 +1,48 @@
+default_node_types:
+  - node_type: spine
+    match_hostnames:
+      - .*spine\d
+  - node_type: l3leaf
+    match_hostnames:
+      - .*l3leaf\d
+
+spine:
+  defaults:
+    loopback_ipv4_pool: 192.168.0.0/24
+    bgp_as: 65000
+  nodes:
+    - name: downlink-ipv4-pool-spine1
+      downlink_pools:
+        - downlink_ipv4_pool: 10.0.1.0/24
+          downlink_interfaces: [Ethernet3, Ethernet4]
+        - downlink_ipv4_pool: 10.0.3.0/24
+          downlink_interfaces: [Ethernet5, Ethernet6]
+      id: 1
+    - name: downlink-ipv4-pool-spine2
+      downlink_pools:
+        - downlink_ipv4_pool: 10.0.2.0/24
+          downlink_interfaces: [Ethernet3, Ethernet4]
+        - downlink_ipv4_pool: 10.0.4.0/24
+          downlink_interfaces: [Ethernet5, Ethernet6]
+      id: 2
+
+l3leaf:
+  defaults:
+    loopback_ipv4_pool: 192.168.1.0/24
+    vtep_loopback_ipv4_pool: 192.168.2.0/24
+    bgp_as: 65000-65100
+    uplink_switches: [downlink-ipv4-pool-spine1, downlink-ipv4-pool-spine2]
+    uplink_interfaces: [Ethernet1, Ethernet2]
+  nodes:
+    - name: downlink-ipv4-pool-l3leaf1
+      id: 1
+      uplink_switch_interfaces: [Ethernet3, Ethernet3]
+    - name: downlink-ipv4-pool-l3leaf2
+      id: 2
+      uplink_switch_interfaces: [Ethernet4, Ethernet4]
+    - name: downlink-ipv4-pool-l3leaf3
+      id: 3
```

Comment:
Let's use some different IDs to ensure we do not pick up the IDs. Like 10, 20, 30, 40. Same for spines.